examples/dswx_s1_sample_runconfig-v3.0.0-er.1.0.yaml

```diff
@@ -0,0 +1,200 @@
+                    hand_file: /home/conda/input_dir/hand.tif
+
+                    reference_water_file: /home/conda/input_dir/reference_water.tif
+
+                    worldcover: /home/conda/input_dir/worldcover.tif
+
+
+            ProductPathGroup:
+                # Path to where output products should be stored
+                # Must correspond to the file system within the Docker container,
+                # and must have write permissions for the User/Group used with
+                # the "Docker run" command
+                OutputProductPath: /home/conda/output_dir
+
+                # Path to a scratch directory for the PGE and SAS to store
+                # intermediate files that will not be needed after PGE execution
+                # completes
+                # Must correspond to the file system within the Docker container,
+                # and must have write permissions for the User/Group used with
+                # the "Docker run" command
+                ScratchPath: /home/conda/output_dir/scratch_dir
+
+            PrimaryExecutable:
+                # Identifier for the PGE executable, should always be DSWX_HLS for
+                # this PGE
+                ProductIdentifier: DSWX_S1
+
+                # Product version specific to output products
+                ProductVersion: 1.0
+
+                # Path to the executable to run, path must be reachable from
+                # within the Docker container (i.e. findable with a "which" command)
+                ProgramPath: python3
+
+                # List of command-line options to use with ProgramPath
+                ProgramOptions:
+                    - /home/conda/dswx-s1-1.0.1/bin/dswx_s1.py
+                    - --full-log-format
+
+                # The Error Code base value added to the offset values to make
+                # error codes unique per-PGE
+                ErrorCodeBase: 100000
+
+                # Path to the Yamale schema used to validate the SAS portion
+                # of the RunConfig
+                # Path should correspond to the file system within the Docker
+                # container, and typically should reference a schema file bundled
+                # with the opera_pge installation directory within the container
+                # Consult the Docker image build scripts for more info
+                SchemaPath: /home/conda/opera/pge/dswx_s1/schema/dswx_s1_sas_schema.yaml
+
+                # Path to the Jinja2 template used to generate the ISO xml
+                # metadata file
+                # Path should correspond to the file system within the Docker
+                # container, and typically should reference a template file bundled
+                # with the opera_pge installation directory within the container
+                # Consult the Docker image build scripts for more info
+                IsoTemplatePath: /home/conda/opera/pge/dswx_s1/templates/OPERA_ISO_metadata_L3_DSWx_S1_template.xml.jinja2
```

Comment:
Since this template doesn't actually exist yet, we should leave this field blank. Otherwise running with this example config could cause a file not found error. If its left empty, I believe the ISO step will be skipped.